package.json

```diff
@@ -1,57 +1,25 @@
 {
-  "name": "bifold-main",
-  "version": "0.0.1",
-  "description": "Aries Mobile Agent React Native is an open source Aries mobile agent app. This is a project that has been created to focus the community's efforts towards a central open source project. Various different organizations and people have expressed interest in a open source community project to help focus efforts to help prevent duplication of work between projects. Aries Bifold is also intended to help get complex or specific use-case projects started faster by cloning and having a basis of an Aries agent.",
-  "main": "entry.js",
+  "name": "aries-bifold",
   "private": true,
-  "directories": {
-    "doc": "docs",
-    "lib": "lib"
-  },
-  "workspaces": {
-    "packages": [
-      "core",
-      "app"
-    ]
-  },
-  "engines": {
-    "node": ">=16.15.0 <18.0.0",
-    "yarn": "^3.3.1"
-  },
-  "resolutions": {
```

Comment:
I found the resolution block required to normalize those dependencies across all packages/workspaces. In addition, it result with having multiple versions of the same dependency which cause problems with react native.

Reply:
Added 👍